php/5.3/Dockerfile

```diff
@@ -26,6 +26,9 @@ RUN echo "Starting ..." && \
     MAKEFLAGS=' -j8' LDFLAGS=-lstdc++ phpenv install ${PHP_VERSION} && \
     phpenv global ${PHP_VERSION} && \
     phpenv rehash && \
+    curl -sSL http://pear.php.net/go-pear.phar -o /tmp/go-pear.phar && \
```

Comment:
Can you use https and check pear does not exist before installing it please?

Reply:
Done